### PR TITLE
rehashing how we operate

### PR DESCRIPTION
**Pull request by ChristinaLK**
Re-organizing and fleshing out some of How We Operate based on trainer discussions.  
Address #821 #909 #929 

Still needs venn diagram (maybe), info about charging, general cleaning up. 

Specific requests: 
- how much of this, if any, should be in the handbook instead, and referenced here? I'm thinking especially of the central vs self-organized workshop pieces. @maneesha 
- is the stuff about regional coordinators correct? should we list their names? @sheraaronhurt 

@karenword @raynamharris

### CODE REVIEW

**Comment by ChristinaLK** [comment]
Ahh shoot, should also address #615 here. may do this now or add it after these major changes.

**Comment by sheraaronhurt** [comment]
@ChristinaLK the verbiage that @raynamharris  created in #909 is great. If you just be sure to link to the page of RC's that would suffice. Thanks!

File: _episodes/21-carpentries.md [hunk 1]
```diff
@@ -150,42 +97,132 @@ and their respective logos
...
+* instructors
+* learners
+
+There are two ways to delegate responsibility for these elements -- a centrally-organized
+or self-organized workshop.
+
+| Component | Centrally-Organized | Self-Organized
+|---------- |-------------------- |-------------- |
+| Location  | The local organizer provides the space and organizes local logistics. | The local organizer provides the space and organizes local logistics. |
+| Instructors | The Carpentries organization finds instructors for the workshop; the local workshop organizer books the instructor's travel and finds local helpers to assist. | The workshop organizer finds both instructors and helpers. |
```
Comment (ChristinaLK) [inline _episodes/21-carpentries.md]: can express travel limitations to the Carpentries.

File: _episodes/21-carpentries.md [hunk 2]
```diff
@@ -18,49 +19,50 @@ keypoints:
...
+approved [Library Carpentry]({{ site.lc_site }})
 as the third official Lesson Program of the Carpentries.
 
-The Carpentries project comprises communities of Instructors, Trainers,
-Maintainers, helpers, and supporters from 
-[Software Carpentry]({{ site.swc_site }}), [Data Carpentry]({{ site.dc_site }}) and [Library Carpentry]({{ site.lc_site }}) 
-who share a mission to teach foundational computational and data science skills. 
+The Carpentries project comprises communities of instructors, trainers,
+maintainers, helpers, and supporters from
+[Software Carpentry]({{ site.swc_site }}), [Data Carpentry]({{ site.dc_site }}) and [Library Carpentry]({{ site.lc_site }})
```
[the remark this answers is not in the document]
Comment (ChristinaLK) [reply _episodes/21-carpentries.md]: I can change it back, I understand the reasoning. 

It was weird to me that instructors/trainers/maintainers were capitalized and helpers wasn't, but I'm guessing that helpers aren't considered official community roles?

File: _episodes/21-carpentries.md [hunk 3]
```diff
@@ -18,49 +19,50 @@ keypoints:
...
-of promoting efficient, shareable, and reproducible research practices. 
+All lesson programs under The Carpentries share the same value
+of promoting efficient, shareable, and reproducible research practices.
 Their aligned missions are accomplished by running accessible, inclusive training workshops; teaching openly available, high-quality, community-developed lessons; and fostering an active, inclusive, diverse instructor community that promotes and models reproducible research as a community norm.
 
 Similarities between Software, Data and Library Carpentry workshops include:
 *   a focus on technical skills.
-*   two-day format taught by volunteer instructors.
+*   an intensive one-day or two-day format taught by volunteer instructors.
 *   a focus on filling gaps in current training for learners.
```
[the remark this answers is not in the document]
Comment (ChristinaLK) [reply _episodes/21-carpentries.md]: 👍

File: _episodes/21-carpentries.md [hunk 4]
```diff
@@ -150,42 +97,132 @@ and their respective logos
 are all trademarked.
 You may only call a workshop a Software Carpentry, Data Carpentry, or Library Carpentry workshop if:
 
-*   it covers the core topics,
-*   at least one instructor is certified,
+*   it covers the core topics (see below),
+*   at least one of the instructors is a certified Carpentries instructor,
```
[the remark this answers is not in the document]
Comment (ChristinaLK) [reply _episodes/21-carpentries.md]: 👍

File: _episodes/21-carpentries.md [hunk 5]
```diff
@@ -150,42 +97,132 @@ and their respective logos
...
+(Software Carpentry, Data Carpentry, or Library Carpentry)
+as they feel able. While instructor badges
+may list a primary lesson program affiliation, instructors are **not** required to certify separately for each. For more
 information, see the description of [the instructor checkout procedure]({{ page.root }}/checkout/).
 
+Note that as long as you have at least one certified instructor, the other
+instructors and helpers at a branded workshop do not have to be certified. We
+rely on the judgment of local organizers to recruit and orient volunteers who
+will embody the Carpentries ethos and practices.
+
```
[the remark this answers is not in the document]
Comment (ChristinaLK) [reply _episodes/21-carpentries.md]: Good idea.

File: _episodes/21-carpentries.md [hunk 6]
```diff
@@ -150,42 +97,132 @@ and their respective logos
...
+Workshop organization is done through Carpentries "regional coordinators".
+This form will alert them to your interest in running a workshop and they will be
+in contact about workshop logistics and requirements. At the moment, there are
+regional coordinators for:
+* North America
+* Europe
+* Africa
+* Australia/New Zealand
+
```
[the remark this answers is not in the document]
Comment (ChristinaLK) [reply _episodes/21-carpentries.md]: I may use @raynamharris suggestion which includes a link to that page.

**Comment by ChristinaLK** [comment]
Addressed most of Karen's suggestions. Moved some of the workshop stuff to the handbook, 
https://github.com/carpentries/handbook/pull/456

Still need to add etherpad, I think. Will probably do that as a separate PR.

**Comment by sheraaronhurt** [comment]
These are great changes. I'd only ask that we make Regional Coordinators a proper noun and capitalize their titles :) Thanks!

**Comment by ChristinaLK** [comment]
@sheraaronhurt should be done.

@karenword @raynamharris is this ready to merge? @brownsarahm is willing to add information about the etherpad to this section, but she should probably work from this copy.

**Comment by brownsarahm** [comment]
should I still wait to address #614 and #615?

**Comment by ChristinaLK** [comment]
@brownsarahm maybe just go ahead and I'll work it in after the fact!

**Comment by raynamharris** [comment]
conflicts resolved!

**Pull request merged**